test/e2e/xtx/sender.go

```diff
@@ -0,0 +1,97 @@
+package xtx
+
+import (
+	"context"
+	"crypto/ecdsa"
+	"math/big"
+
+	"github.com/omni-network/omni/lib/errors"
+	"github.com/omni-network/omni/lib/log"
+	"github.com/omni-network/omni/lib/txmgr"
+	"github.com/omni-network/omni/test/e2e/netman"
+
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+)
+
+type Sender struct {
```

Comment:
lets not export this, can also rename to `wrapper` or `instance` or `single` or something like that